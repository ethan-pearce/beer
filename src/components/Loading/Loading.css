.loadingText {
    font-size: 40px;
}

.loadingImage {
    width: 300px;
}