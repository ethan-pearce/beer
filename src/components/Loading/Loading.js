import './Loading.css';

export const Loading = () => {
    return (
        <>
            <div className={'loadingText'}>Loading ....</div>
            <img className={'loadingImage'} src={'https://media4.giphy.com/media/dt91SdhwBusuZalJkG/giphy.gif?cid=ecf05e4769y62aeey7sw7k7bcuiu66q3qyjobrh2exgafgl6&rid=giphy.gif'} />
        </>
    )
}
export default Loading;