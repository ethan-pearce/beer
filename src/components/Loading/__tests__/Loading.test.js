import React from 'react'
import { render } from '@testing-library/react';
import Loading from '../Loading'

describe('Loading Component', () => {
    it('should render', () => {
        const { container } = render(<Loading />)
        expect(container.getElementsByClassName('loadingText')[0].innerHTML).toBe('Loading ....')
        expect(container).toMatchSnapshot()
    })
})