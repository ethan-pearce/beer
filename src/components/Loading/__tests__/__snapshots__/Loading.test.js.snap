// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading Component should render 1`] = `
<div>
  <div
    class="loadingText"
  >
    Loading ....
  </div>
  <img
    class="loadingImage"
    src="https://media4.giphy.com/media/dt91SdhwBusuZalJkG/giphy.gif?cid=ecf05e4769y62aeey7sw7k7bcuiu66q3qyjobrh2exgafgl6&rid=giphy.gif"
  />
</div>
`;
